config/core/test-configs.yaml

```diff
@@ -2432,6 +2443,8 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - kselftest-futex
+      - kselftest-livepatch
```

Comment:
We don't have that many `rock2` board online, so I'm not quite sure about this. Only one online in Collabora's lab at the moment. It's probably best to drop this now or we'll just be filling the queue with jobs that can't be run.